docs/Cloud_integrations/webhooks.md

```diff
@@ -68,3 +71,6 @@ To validate the webhook parameters, it is possible to **test the event handler**
 Click **Next step** and **Add event handler** to create the event handler.
 
 ![Active webhook](images/active-webhook.png "Active webhook")
+
+!!! warning
+    An event handler is automatically deactivated after several failed attempts. Manually reactivate the event handler to continue using the data integration.
```

Comment:
I think you might add `several consecutive failed attempts` - for example, if the handler fails every other time, it won't be disabled.